torch/_inductor/codegen/cpp.py

```diff
@@ -3799,10 +3783,27 @@ def _get_outer_loop_fusion_depth(self, node1, node2):
                 return outer_loop_fusion_depth
         return DISABLE_OUTER_LOOP_FUSION
 
+    def is_vertical_outer_loop_fusion(self, node1, node2):
```

Comment:
Should this be `can_fuse_vertically_outer_loop`, in line with `can_fuse_horizontal`?

Reply:
Thanks and changed.